index.php

```diff
@@ -54,7 +53,7 @@
                     ?>
                 </div>
                 <button type="submit" id="download" class="suggested-action">Download Freya</button>
-                <p class="small-label"><?php echo round($filesize / 1000 / 1000); ?> MB (for PC or Mac)</p>
+                <p class="small-label">893 MB (for PC or Mac)</p>
```

Comment:
This is a workaround that works for now, but if we're going to want updated ISOs, we'll have to manually keep updating this page. Are we cool with that?